server/views/challenges/showZiplineOrBasejump.jade

```diff
@@ -8,11 +8,10 @@ block content
                     | &nbsp;
                     i.ion-checkmark-circled.text-primary(title="Completed")
             hr
-            ol
+            ul
                 for step, index in description
                     .row.checklist-element(id="#{dashedName + index}")
-                        .col-xs-3.col-sm-1.col-md-2.padded-ionic-icon.text-center
-                            input(type='checkbox' class='challenge-list-checkbox')
+                        .col-xs-3.col-sm-1.col-md-2
```

Comment:
@atjonathan not "weird" i should have said "it look better with that line"
btw screenshot is pic[1] and on changing on changing col-9  => col-12 it looks like pic[2]. Tell me which one is better out of three.

pic[1]
![screenshot from 2016-06-02 01-02-59](https://cloud.githubusercontent.com/assets/11391128/15723004/20776d28-285e-11e6-9913-9eb0ff4088bd.png)
pic[2]
![screenshot from 2016-06-02 01-06-49](https://cloud.githubusercontent.com/assets/11391128/15723123/a48b6560-285e-11e6-97a5-b9092c23b979.png)